src/lib.rs

```diff
@@ -153,6 +153,26 @@ pub enum MemberOrigin {
     Offset(u32),
 }
 
+/// The interpolation qualifier of a binding or struct field.
+#[derive(Clone, Copy, Debug, PartialEq)]
+#[cfg_attr(feature = "serialize", derive(Serialize))]
+#[cfg_attr(feature = "deserialize", derive(Deserialize))]
+pub enum Interpolation {
+    /// Indicates that linear, non-perspective, correct
```

Comment:
we should have `Perspective` here as well, explicitly

Reply:
Perspective would be default interpolation (aka, None)? I added all the kinds of interpolation supported by spirv.

Reply:
The IR doesn't need defaults, nobody writes it by hand. I want it to be explicit and clear.